services/workflows/baseWorkflow.js

```diff
@@ -1,35 +1,42 @@
 "use strict";
 
+const workflowError = require("../../errors/workflowError");
+const container = require("../../containerConfig");
 const jWorkflow = require("jWorkflow");
-const _ = require("lodash");
+const config = require("config");
```

Comment:
require("lodash"); -> check tree shaking

Reply:
fixed in db7688b